v1/api_client.go

```diff
@@ -1,7 +1,7 @@
 /*
  * JumpCloud APIs
  *
- * V1 & V2 versions of JumpCloud's API. The previous version of JumpCloud's API. This set of endpoints allows JumpCloud customers to manage commands, systems, & system users.
+ *  JumpCloud's V1 API. This set of endpoints allows JumpCloud customers to manage commands, systems, & system users.
```

Comment:
Is the space supposed to be there at the beginning of the line?

Reply:
that seems just to be how Stoplight renders that section. no space to remove.